torch/jit/_freeze.py

```diff
@@ -10,7 +10,7 @@
 from torch.jit._script import RecursiveScriptModule, ScriptModule
 
 
-def freeze(mod, preserved_attrs: Optional[List[str]] = None):
+def freeze(mod, preserved_attrs: Optional[List[str]] = None, optimize: bool = True):
```

Comment:
It might be easier to land this in two steps: 1) add a new flag with the new functionality, but disabled by default, 2) flip the default. This way if something breaks, only the flag switch would be reverted.